cryptofeed/exchange/bitfinex.py

```diff
@@ -63,6 +68,14 @@ def _parse_symbol_data(cls, data: list, symbol_separator: str) -> Tuple[Dict, Di
                 norm = norm[:3] + symbol_separator + norm[3:]
             ret[norm] = "t" + p
 
+        for f in perpetuals:
+            norm = f.replace('BCHN', 'BCH')
+            norm = norm.replace('UST', 'USDT')
```

Comment:
`UST != USDT`, former is the Terra blockchain one

and is `BCHN` actually bitcoin cash on Bitfinex?

Reply:
This is just a mapping for the standardised symbol that cryptofeed maintains - so I don't think we need to worry about it, as it doesn't affect our reference data, and how we subscribe.